src/app/cli/src/coda_process.ml

```diff
@@ -59,76 +56,76 @@ let disconnect (conn, proc, _) =
   let%map (_ : Unix.Exit_or_signal.t) = Process.wait proc in
   ()
 
-let peers_exn (conn, proc, _) =
+let peers_exn (conn, _, _) =
```

Comment:
It might make the intended use of these functions clearer if this argument is still named (`_proc` would make sense).